stattutor/static/js/Initialize_CTATXBlock.js

```diff
@@ -0,0 +1,113 @@
+/**
+ * Called by edX to initialize the xblock.
+ * @param runtime - provided by EdX
+ * @param element - provided by EdX
+ */
+function Initialize_CTATXBlock(runtime,element) 
```

Comment:
It looks like you massively reformatted this file since I saw it last.
Why does it follow a different coding convention than the rest of your JS?

https://google.github.io/styleguide/jsguide.html
http://edx.readthedocs.io/projects/edx-developer-guide/en/latest/style_guides/javascript_guidelines.html